### PR TITLE
.getEventStream method compatible with new product event stream endpoints

### PR DESCRIPTION
**Pull request by jme783**
Support the following new routes (all have tests):

```
/v1/products/:productIdOrSlug/events
/v1/products/:productIdOrSlug/events/:event_name
/v1/products/:productIdOrSlug/devices/:orgDeviceId/events
/v1/products/:productIdOrSlug/devices/:orgDeviceId/events/:event_name
```

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+	"root": true,
```
Comment (monkbroc) [inline .eslintrc]: Why lint only the current directory? `npm run lint` passes for me without this.

Comment (brycekahle) [inline .eslintrc]: This prevents `eslint` from searching above this directory for configuration. It doesn't actually limit it to the current directory, but anything below the current directory.

**Comment by monkbroc** [comment]
@brycekahle Should we only update the `dist` folder right before a release to avoid merge conflicts in PRs?

File: .eslintrc (hunk 1 above)
Comment (monkbroc) [inline .eslintrc]: OK. Thanks. So we leave this in then?

Comment (brycekahle) [inline .eslintrc]: Yes

**Comment by brycekahle** [comment]
Yeah, probably. Also not updating docs until a release is best.

**Comment by monkbroc** [comment]
Agreed Bryce. I noticed that updating the docs changes all the hashes so doing it not too frequently is best.

The actual change is good. @jme783, I'll squash the commits and merge.

**Comment by jme783** [comment]
Sorry about running the build. Didn't realize that this was done automatically

**Comment by monkbroc** [comment]
I pulled the rug under Travis' feet by deleting the branch before the checks finished...

**Pull request merged**